Microsoft.Toolkit.Uwp.UI.Controls/BladeView/BladeView.Properties.cs

```diff
@@ -24,24 +24,24 @@ namespace Microsoft.Toolkit.Uwp.UI.Controls
     /// A container that hosts <see cref="BladeItem"/> controls in a horizontal scrolling list
     /// Based on the Azure portal UI
     /// </summary>
-    public partial class BladeControl
+    public partial class BladeView
     {
         /// <summary>
         /// Identifies the <see cref="Blades"/> dependency property.
         /// </summary>
         [Deprecated("This property has been replaced with the Items property of the control. It is no longer required to place content within the Blades property.", DeprecationType.Deprecate, 1)]
-        public static readonly DependencyProperty BladesProperty = DependencyProperty.Register(nameof(Blades), typeof(IList<Blade>), typeof(BladeControl), new PropertyMetadata(null, OnBladesChanged));
+        public static readonly DependencyProperty BladesProperty = DependencyProperty.Register(nameof(Blades), typeof(IList<Blade>), typeof(BladeView), new PropertyMetadata(null, OnBladesChanged));
```

Comment:
perhaps a stupid idea: why not moving this to the BladeControl ?this way we could avoid using comments to hide deprecated warnings